src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,68 @@
+.menu {
+  background-color: $colorRed;
+
+  &__content {
+    box-sizing: border-box;
+    padding-top: $headerTopPaddingMobile;
+    height: 100vh;
+
+    @include onTablet {
+      padding-top: 32px;
+    }
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 40px;
+  }
+
+  &__list {
+    padding: 0;
+    list-style: none;
+    margin-bottom: 28px;
+  }
+
+  &__item {
+    color: $colorWhite;
+    margin-bottom: 28px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    color: $colorWhite;
+    text-decoration: none;
+    @include propertyForInlineCenter(22px);
+    letter-spacing: 2px;
+    text-transform: uppercase;
+  }
+
+  &__call {
+    display: block;
```

Comment:
all elements in flex container block by default.

Reply:
These elements are not elements of flex-container